packages/components/psammead-section-label/src/index.stories.jsx

```diff
@@ -16,6 +16,10 @@ storiesOf('Components|SectionLabel', module)
           script={script}
           dir={dir}
           bar={boolean('show bar?', true)}
+          hideSectionHeader={boolean(
+            'hide first section header when width less than 600px?',
+            true,
```

Comment:
Can this default to false please? If the user loads the storybook with the dev tools open etc, the preview window might be <600 and it makes it look like the component didnt load. If the user has to toggle to hide it, they know why its not showing :)